connect-inject/cleanup_resource_ent_test.go

```diff
@@ -124,7 +125,8 @@ func TestReconcile_ConsulNamespaces(t *testing.T) {
 
 			// Run Reconcile.
 			cleanupResource.reconcile()
-
+			// FIXME: timing issue here popping up also in main branch
+			time.Sleep(time.Second * 1)
```

Comment:
Temp fix until this one gets merged.
https://github.com/hashicorp/consul-k8s/pull/449/files